Remove global Mn, BB

denar90 opened the pull request:
Added global $, Mn, Bb for skeleton

scott-w commented on brunch/brunch-config.js:
```diff
@@ -12,6 +12,13 @@ module.exports = {
+    globals: {
+      $: 'jquery',
+      Marionette: 'backbone.marionette',
+      Backbone: 'backbone.marionette/node_modules/backbone'
```
Why use globals instead of explicitly importing the variables? I appreciate `$` can be unavoidable.

denar90 replied:
I just wanted to simplify app files using global Mn. Most of app files will use $, Mn, Bb and it's became annoying adding each time in top of them smth like `import Marionette from 'backbone.marionette`. That's why I proposed it)
What do you think, is it useless? Should we live it as it is?

denar90 commented:
@samccone @scott-w do I have to close this PR or smth?

scott-w commented:
If you're ok with it as-is, then feel free.

Personally, I prefer forcing imports everywhere - it makes it clear what you're using and where.

denar90 commented:
Should I remove Mn and stuff from global scope in browserify and webpack examples? To have consistent way everywhere?

scott-w commented:
A consistent approach would be best. If anyone else has a preference for global/non-global, feel free to chip in.

denar90 commented:
I've updated PR. I added travis to check if build for `browserify`, `brunch`, `webpack` going well
But it is needed to add "secure key" to `travis.yaml` to make it works.

scott-w commented:
:+1:

denar90 commented:
@samccone can you look at this PR?

ahumphreys87 commented:
:+1:
